src/grid/PlanGridView.php

```diff
@@ -110,6 +110,15 @@ public function columns(): array
                     ],  
                 ]),
             ],
+            'parent_plan' => [
+                'format' => 'raw',
+                'value' => function (Plan $model): string {
+                    if ($model->hasParent()) {
+                        return str_replace('&nbsp;', '', strip_tags($model->parent->name));
```

Comment:
`strip_tags` [does not](https://stackoverflow.com/questions/5788527/is-strip-tags-vulnerable-to-scripting-attacks) protect against XSS.

Use `Html::encode()`